src/app/test/db/mongo/util_test.py

```diff
@@ -19,11 +19,13 @@ def model_mocks(monkeypatch):
     role_mock = Mock()
     job_mock = Mock()
     principal_mock = Mock()
+    event_mock = Mock()
```

Comment:
I'm guessing the updates to this file are accidental leftovers after you moved the code and tests.  You'll want to clean these up.